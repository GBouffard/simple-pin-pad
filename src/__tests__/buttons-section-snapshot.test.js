import React from 'react';
import ButtonsSection from '../components/molecules/buttons-section';
import renderer from 'react-test-renderer';

describe('ButtonsSection - ', () => {
  const component = renderer.create(
    <ButtonsSection />
  );

  const tree = component.toJSON();

  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});