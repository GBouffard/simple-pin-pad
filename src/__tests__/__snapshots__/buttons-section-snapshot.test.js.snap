// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonsSection -  renders correctly 1`] = `
<section
  className="App__button-section"
>
  <button
    className="App__pin-pad-button"
    onClick={[Function]}
  >
    1
  </button>
  <button
    className="App__pin-pad-button"
    onClick={[Function]}
  >
    2
  </button>
  <button
    className="App__pin-pad-button"
    onClick={[Function]}
  >
    3
  </button>
  <button
    className="App__pin-pad-button"
    onClick={[Function]}
  >
    4
  </button>
  <button
    className="App__pin-pad-button"
    onClick={[Function]}
  >
    5
  </button>
  <button
    className="App__pin-pad-button"
    onClick={[Function]}
  >
    6
  </button>
  <button
    className="App__pin-pad-button"
    onClick={[Function]}
  >
    7
  </button>
  <button
    className="App__pin-pad-button"
    onClick={[Function]}
  >
    8
  </button>
  <button
    className="App__pin-pad-button"
    onClick={[Function]}
  >
    9
  </button>
  <button
    className="App__pin-pad-button App__last-button"
    onClick={[Function]}
  >
    0
  </button>
</section>
`;
